server/models/List.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose");
+
+const listSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  products: [{
+    type: mongoose.Schema.ObjectId,
+    ref: 'products'
+  }],
+  userIds: [{
```

Comment:
great! just a simple thing: it would be good to know who owns a list, to whom notifications should be sent and who has the right to modify/delete a list or products in a list? I suggest adding a reference to the user who creates the list. 

PS: from your POV, it would be nice to:
- add a field referencing the user who creates the list
- add this user to usersIds, this way we send notifications to all users who follow this list

Reply:
I already added a creator field below. do u think it's ok to duplicate data (the creator id would be in 2 places) for simplifying the controllers? @dproc96 @christian-barnabe-chabi

Reply:
> I already added a creator field below. do u think it's ok to duplicate data (the creator id would be in 2 places) for simplifying the controllers? @dproc96 @christian-barnabe-chabi

See, you're right. oh my bad, it's okay! You know, I was using the previous model to implement part of the task-queues, and I was looking for the `userId` field and I didn't pay attention to the `creator` 😂.